nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -302,8 +302,7 @@ def submit_job(self, args, api: AdminAPISpec):
         # zip the data
         data = zip_directory_to_bytes(self.upload_dir, folder_name)
 
-        rel_path = os.path.relpath(full_path, self.upload_dir)
-        folder_name = remove_leading_dotdot(rel_path)
+        folder_name = split_path(full_path)[1]
```

Comment:
What if the full_path is "/home/someuser/somefolder/job"?

Then folder_name becomes "someuser"?

Reply:
Upon reading: https://docs.python.org/3/library/os.path.html#os.path.split

I figure it will return the last piece.